util/puppet_agent_mgr.rb

```diff
@@ -207,11 +207,12 @@ def valid_name?(name)
       end
 
       # validates arguments and returns the CL options to execute puppet
-      def create_common_puppet_cli(noop=nil, tags=[], environment=nil,
+      def create_common_puppet_cli(noop=nil, tags=[], skip_tags=[], environment=nil,
                                    server=nil, splay=nil, splaylimit=nil,
                                    ignoreschedules=nil, use_cached_catalog=nil)
```

Comment:
I though it was preferable to have `tags` and `skip_tags` next to each other, at the cost of quite some changes that could be avoided otherwise.